frontend/src/app/style-guide/style-guide.component.html

```diff
@@ -53,6 +53,22 @@ <h2>Colors</h2>
     <p class="demo-fs-red-dark">$fs-red-dark</p>
     <p class="demo-fs-red-darker">$fs-red-darker</p>
 
+    <h2>Accordion</h2>
```

Comment:
This this is no longer being used in the rules, do we want to remove?

Reply:
Removed